### PR TITLE
Add missing `toHaveHTML` to matcher

### PR DESCRIPTION
**Pull request by LukasHeimann**
Fixes https://github.com/webdriverio/expect-webdriverio/issues/1413

### CODE REVIEW

**Comment by christian-bromann** [comment]
Thanks for the patch, mind signing the CLA?

**Comment by LukasHeimann** [comment]
@christian-bromann CLA is signed!

**Review by christian-bromann — APPROVED** [review]
Thanks a lot 👍

File: test/matchers.test.ts [hunk 1]
```diff
@@ -43,6 +43,8 @@ test('matchers', () => {
         'toHaveId',
         'toHaveSize',
         'toHaveElementProperty',
+        'toHaveHTML',
+        'toHaveHTMLContaining',
```
Comment (christian-bromann) [inline test/matchers.test.ts:47]:
```suggestion
        'toHaveHTML',
        'toHaveHTMLContaining',
        'toHaveId',
        'toHaveSize',
        'toHaveElementProperty',
```

**Pull request merged**